.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "useTabs": false,
```

Comment:
Can be removed as it's the default value.

https://prettier.io/docs/en/options.html#tabs

Reply:
yep